basicswap/bin/prepare.py

```diff
@@ -114,16 +117,14 @@
     "dogecoin": (DOGECOIN_VERSION, DOGECOIN_VERSION_TAG, ("tecnovert",)),
 }
 
-disabled_coins = [
-    "navcoin",
-    "namecoin",  # Needs update
-]
+disabled_coins = ["navcoin"]
 
 expected_key_ids = {
     "tecnovert": ("13F13651C9CF0D6B",),
     "thrasher": ("FE3348877809386C",),
     "laanwj": ("1E4AED62986CD25D",),
     "JeremyRand": ("2DBE339E29F6294C",),
+    "RoseTuring": ("9FE3BFDDA6C53495",),
```

Comment:
I would suggest using the full fingerprints rather than truncated. There's a long history of people using collision-vulnerable hashes in contexts where they *thought* they weren't dangerous, and it turned out they were mistaken. [This slide deck](https://web.archive.org/web/20241113093105/https://marc-stevens.nl/research/hashclash/rogue-ca/downloads/md5-collisions-1.0.pdf) is routinely used as an example of this in crypto 101 at my alma mater.

(I believe I do see a practical attack that could happen when using collision-vulnerable hashes in this context. I'm not going to say what the attack is, but the fact that I can't trivially convince myself that the attack doesn't work is already a problem -- at best, it makes audits more expensive for no good reason.)

Reply:
Will update all in a follow-up pr